### DISCUSSION TITLE
Biscuit Theme Request

### DISCUSSION THREAD

**Discussion by guzman109** [NvChad/NvChad#2862] [Ideas]
Not sure if this is the correct place but I was wondering if someone had a confit for Biscuit Theme (https://github.com/Biscuit-Theme/biscuit)?

@siduck anyway this could be added to nvchad?

**Comment by siduck** [comment]
@guzman109 this theme has been banned from NvChad, along with catpuccin ofcourse

**Reply by guzman109** [reply]
Bummer but understandable. Btw why do they get banned? Do they just not let you use it? Is it the license?

**Reply by siduck** [reply]
@guzman109  there are some other reasons which i cant tell, if I tell it then i wont exist on github
